platform/macos/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 This version of the Mapbox macOS SDK corresponds to version 3.4.1 of the Mapbox iOS SDK. The two SDKs have very similar feature sets. The main differences are the lack of user location tracking and annotation views. Some APIs have been adapted to macOS conventions, particularly the use of NSPopover for callout views.
 
 * Fixed an issue causing MGLMapView’s `camera`’s `heading` to be set to a negative value, indicating an undefined heading, when the map view faces northwest. The heading is now wrapped to between zero and 360 degrees, for consistency with MGLMapView’s `direction` property. ([#7724](https://github.com/mapbox/mapbox-gl-native/pull/7724))
+* Fixed an issue causing MGLMapView flash of black. ([#7859](https://github.com/mapbox/mapbox-gl-native/pull/7859))
```

Comment:
This PR doesn’t change anything about the macOS SDK, since MGLMapView has separate implementations on iOS and macOS. The behavior is already correct on macOS, so you can remove this blurb.